tests/pytype_test.py

```diff
@@ -131,14 +135,43 @@ def find_stubs_in_paths(paths: Sequence[str]) -> list[str]:
     return filenames
 
 
+def get_missing_modules(files_to_test: Sequence[str]) -> Iterable[str]:
+    """Gets module names provided by typeshed-external dependencies.
+
+    Some typeshed stubs depend on dependencies outside of typeshed. Since pytype
+    isn't able to read such dependencies, we instead declare them as "missing"
+    modules, so that no errors are reported for them.
+    """
+    stub_distributions = set()
+    for fi in files_to_test:
+        parts = fi.split(os.sep)
+        try:
+            idx = parts.index("stubs")
+        except ValueError:
+            continue
+        stub_distributions.add(parts[idx + 1])
+    missing_modules = set()
+    for distribution in stub_distributions:
+        for pkg in utils.read_dependencies(distribution).external_pkgs:
+            # See https://stackoverflow.com/a/54853084.
```

Comment:
Tiny nit to fix the link on github
```suggestion
            # See https://stackoverflow.com/a/54853084
```